src/components/Correction/CompletingParty.vue

```diff
@@ -1,92 +0,0 @@
-<template>
```

Comment:
Note that there is a **common** CompletingParty component that is still used by Change, Conversion and Firm Correction filings.